.github/workflows/nightly.yml

```diff
@@ -0,0 +1,41 @@
+name: "Nightly Build"
+on:
+  schedule:
+    - cron: "10 * * * *"
+
+jobs:
+  check-date:
```

Comment:
This job is supposed to make it possible to stop running the job if there was no change to the branch in the last 24 hours.